app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -51,40 +80,77 @@ class SearchActivity : AppCompatActivity() {
         }
         inputEditText.addTextChangedListener(simpleTextWatcher)
 
-        val recyclerView = findViewById<RecyclerView>(R.id.recyclerView)
-        recyclerView.layoutManager = LinearLayoutManager(this)
-
-        //Track 1
-        val name1 = resources.getString(R.string.track1_name)
-        val time1 = resources.getString(R.string.track1_time)
-        val author1 = resources.getString(R.string.track1_author)
-        val imgSrc1 = resources.getString(R.string.track1_imageSource)
-        //Track 2
-        val name2 = resources.getString(R.string.track2_name)
-        val time2 = resources.getString(R.string.track2_time)
-        val author2 = resources.getString(R.string.track2_author)
-        val imgSrc2 = resources.getString(R.string.track2_imageSource)
-        //Track 3
-        val name3 = resources.getString(R.string.track3_name)
-        val time3 = resources.getString(R.string.track3_time)
-        val author3 = resources.getString(R.string.track3_author)
-        val imgSrc3 = resources.getString(R.string.track3_imageSource)
-        //Track 4
-        val name4 = resources.getString(R.string.track4_name)
-        val time4 = resources.getString(R.string.track4_time)
-        val author4 = resources.getString(R.string.track4_author)
-        val imgSrc4 = resources.getString(R.string.track4_imageSource)
-        //Track 5
-        val name5 = resources.getString(R.string.track5_name)
-        val time5 = resources.getString(R.string.track5_time)
-        val author5 = resources.getString(R.string.track5_author)
-        val imgSrc5 = resources.getString(R.string.track5_imageSource)
-
-        val trackList: List<Track> = listOf(Track(name1, author1, time1, imgSrc1), Track(name2, author2, time2, imgSrc2), Track(name3, author3, time3, imgSrc3), Track(name4, author4, time4, imgSrc4), Track(name5, author5, time5, imgSrc5) )
-
-        val trackAdapter = TrackAdapter(trackList)
-        recyclerView.adapter = trackAdapter
-
+        inputEditText.setOnEditorActionListener { _, actionId, _ ->
+            if (actionId == EditorInfo.IME_ACTION_DONE) {
+                trackService.search(inputEditText.text.toString()).enqueue(object :
+                    Callback<TrackResponse> {
+                    override fun onResponse(call: Call<TrackResponse>,
+                                            response: Response<TrackResponse>
+                    ) {
+                        if (response.code() == 200) {
```

Comment:
```suggestion
                        if (response.isSuccessfull) {
```
Лучше написать так, чтобы не хардкодить коды ответов http запросов